docs/installation/centos.md

```diff
@@ -53,7 +53,7 @@ package manager.
 
 3. Add the yum repo.
 
-        $ cat >/etc/yum.repos.d/docker.repo <<-EOF
+        $ echo | sudo tee /etc/yum.repos.d/docker.repo <<-EOF
```

Comment:
ha, sure enough. it fit the existing pattern and worked i guess. it does appear superfluous.